src/einsteinpy/rays/shadow.py

```diff
@@ -88,6 +114,10 @@ def _intensity_from_event_horizon(self):
         return k1
 
     def smoothen(self, points=500):
+        '''
+        Returns the interpolated values for the intensities for smoothening of the plot
```

Comment:
It does not return anything BTW.
```suggestion
        Sets the interpolated values for the intensities for smoothening of the plot
```